indico/web/client/js/react/util/html.js

```diff
@@ -26,10 +26,18 @@ export function toClasses(...params) {
  * It takes a function which will be called with `resolve, reject` and render
  * the actual component.
  */
-export function injectModal(renderFunc) {
+export function injectModal(renderFunc, position) {
```

Comment:
```suggestion
export function injectModal(renderFunc, position = undefined) {
```

it's nicer to be clear that this argument is optional

Reply:
Done